test/lib/Elastica/Test/Query/MoreLikeThisTest.php

```diff
@@ -2,6 +2,8 @@
 namespace Elastica\Test\Query;
 
 use Elastica\Document;
+use Elastica\Filter\Bool;
```

Comment:
You have to use BoolFilter here to make it compatible with PHP 7: https://github.com/ruflin/Elastica/blob/master/lib/Elastica/Filter/BoolFilter.php